src/main/java/unrc/dose/Comment.java

```diff
@@ -8,10 +12,52 @@
 * description     :varchar(300)    not null
 * challenge_id    :integer     not null
 * user_id         :integer    not null
+* comment_id      :integer
 **/
-package unrc.dose;
+/**
+*@author Celi Yanina, de Prada Hernan, Goldenberg Erika
+**/
+public class Comment extends Model {
 
-import org.javalite.activejdbc.Model;
+  /**
+  *Create a new comment.
+  *@param title this is the comment's title
+  *@param description this the comment's body
+  *@param challenge_id this is the challenge's id which is commented
+  *@param user_id this is the user's id who commented
+  **/
 
-public class Comment extends Model {
+  public void createComment (String title,String description, int challenge_id, int user_id ){
+    this.set("title",title);
+    this.set("description",description);
+    this.set("challenge_id", challenge_id);
+    this.set("user_id", user_id);
+    this.saveIt();
+  }
+  /*
+  *isResponse
+  *@param comment_id this is comments or response to evaluate
+  *@return return if the selected comment_id si a comment or a response.
+  **/
+  public boolean isResponse(int comment_id){
+    Comment c = Comment.findById(comment_id);
+    return ((c.getInteger("comment_id")) != (null));
+  }
+  /**
+  *Create a new Response.
+  *@param description this the response's body
+  *@param challenge_id this is the challenge's id which is commented
+  *@param user_id this is the user's id who responded to a comment
+  **/
+  public void createResponse (String description, int user_id, int comment_id) throws NullPointerException{
```

Comment:
This method should be static since without it the method requires an instance of `Comment` to be used, this method should also return a `Comment` object (the created response).

`NullPointerException` is an unchecked exception which are not declared in the `throws` section.